README.md

```diff
@@ -14,66 +9,44 @@ This JavaScript library is part of the [Microsoft Teams developer platform](http
 
-  TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the packages/teams-js directory.
-
-## Installation
-
-To install the latest 2.0 preview version:
-
-### npm
+TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the `packages/teams-js` directory.
 
-`npm install --save @microsoft/teams-js@next`
+This JavaScript library is part of the [Microsoft Teams developer platform](https://docs.microsoft.com/en-us/microsoftteams/platform/overview?view=msteams-client-js-beta). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-beta).
 
-### yarn
+[![Build Status](https://travis-ci.org/OfficeDev/microsoft-teams-library-js.svg?branch=2.0-preview)](https://travis-ci.org/OfficeDev/microsoft-teams-library-js)
+[![Coverage Status](https://coveralls.io/repos/github/OfficeDev/microsoft-teams-library-js/badge.svg?branch=2.0-preview)](https://coveralls.io/github/OfficeDev/microsoft-teams-library-js?branch=2.0-preview)
 
-`yarn add @microsoft/teams-js@next`
+# Packages
 
-### Production
+Contain the core exports for the repository.
 
-You can access [these files on unpkg](https://res.cdn.office.net/teams-js/2.0.0-beta.2/js/MicrosoftTeams.min.js), download them, or point your package manager to them.
+### [teams-js](./packages/teams-js)
 
-## Usage
+Used to integrate custom services and applications with Teams, Outlook, and Office.
 
-### As a package
+### [Bundle Size Tools](./packages/bundle-size-tools/README.md)
 
-Install either using npm or yarn.
+Package used to measure the difference in size between two recent commits.
 
-**If you are using any dependency loader** such as [RequireJS](http://requirejs.org/) or [SystemJS](https://github.com/systemjs/systemjs) or module bundler such as [browserify](http://browserify.org/), [webpack](https://webpack.github.io/), you can use `import` syntax to import specific modules. For e.g.
-
-```typescript
-import { app } from '@microsoft/teams-js';
-```
-
-### As a script tag
-
-Reference the SDK inside of your `.html` page using:
+---
 
-```html
-<!-- Microsoft Teams JavaScript API (via CDN) -->
-<script
-  src="https://res.cdn.office.net/teams-js/2.0.0-beta.2/js/MicrosoftTeams.min.js"
-  integrity="sha384-Q2Z9S56exI6Oz/ThvYaV0SUn8j4HwS8BveGPmuwLXe4CvCUEGlL80qSzHMnvGqee"
-  crossorigin="anonymous"
-></script>
+# Apps
 
-<!-- Microsoft Teams JavaScript API (via npm) -->
-<script src="node_modules/@microsoft/teams-js@2.0.0-beta.2/dist/MicrosoftTeams.min.js"></script>
+The apps folder contains applications used to test various aspects of the sdk.
 
-<!-- Microsoft Teams JavaScript API (via local) -->
-<script src="MicrosoftTeams.min.js"></script>
-```
+### [Bundle Analysis App](./apps/bundle/analysis-app/README.md)
 
-### Dependencies
+This application is responsible for testing the size of the final sdk bundle.
```

Comment:
Nit: keeping consistent with the structure of the other summaries, can you please change this to:
"Application responsible for testing the size of the final SDK bundle."